index.js

```diff
@@ -50,10 +50,16 @@ module.exports = (args, flags) => {
         fs.mkdirSync(flags.outputDir);
       }
 
-      await fs.writeFile(`${flags.outputDir}/hubble-data.json`, prettyJSON(mapping));
+      const fsErrorHandler = (err) => {
+        if (err) {
+          throw new Error(err);
```

Comment:
Maybe put a console.error here too so stdout receives it when using the binary in other applications.

```suggestion
	      console.error('Error trying to write to file:', err);
          throw new Error(err);
```

Reply:
Done. 👍